mmdet/core/anchor/anchor_generator.py

```diff
@@ -213,12 +213,14 @@ def _meshgrid(self, x, y, row_major=True):
         else:
             return yy, xx
 
-    def grid_priors(self, featmap_sizes, device='cuda'):
+    def grid_priors(self, featmap_sizes, dtype=torch.float32, device='cuda'):
```

Comment:
Suggest that add the argument `dtype` after `device` in the function definition.

Reply:
In pytorch, dtype is in the former of device. We mightneed to be consistent with that